src/profile-logic/transforms.js

```diff
@@ -905,6 +937,76 @@ const FUNC_MATCHES = {
   },
 };
 
+export function collapseFunctionSubtree(
+  thread: Thread,
+  funcToCollapse: IndexIntoFuncTable
+): Thread {
+  const { stackTable, frameTable, samples } = thread;
+  const oldStackToNewStack: Map<
+    IndexIntoStackTable | null,
+    IndexIntoStackTable | null
+  > = new Map();
+  oldStackToNewStack.set(null, null);
```

Comment:
nit: I'd appreciate a comment to explain why you insert this (null, null) association.

Reply:
I updated it here, and anywhere else we use this practice.